src/v1/internal/util.js

```diff
@@ -52,10 +73,17 @@ function assertString(obj, objName) {
 
 function assertCypherStatement(obj) {
   assertString(obj, 'Cypher statement');
-  if (obj.trim().length == 0) {
+  if (obj.trim().length === 0) {
     throw new TypeError('Cypher statement is expected to be a non-empty string.');
   }
-  return obj;
+}
+
+function assertQueryParameters(obj) {
+  if (!isObject(obj) && Boolean(obj)) {
```

Comment:
`isObject(obj)` already includes the condition `Boolean(obj)`, maybe just remove `&& Boolean(obj)` here?